chargeamps/external.py

```diff
@@ -226,3 +239,13 @@ async def reboot(self, charge_point_id) -> None:
         """Reboot chargepoint"""
         request_uri = f"/api/{API_VERSION}/chargepoints/{charge_point_id}/reboot"
         await self._put(request_uri, json="{}")
+
+    async def get_logged_in_user(self) -> User:
+        """Get authenticated user info"""
+        user_id = self._user["id"]
+
+        request_uri = f"/api/{API_VERSION}/users/{user_id}"
+        response = await self._get(request_uri)
+        payload = await response.json()
+
+        return User.model_validate(payload)
```

Comment:
Since the `user` class variable is of type User, and `id` of that class is not optional, there will always be an `id` there as long as a user has been assigned. Will add a check for if there is currently a user logged in though.
Regarding the error handling, this follows the same design pattern as elsewhere in the module. While good to have, I think it is better to keep the design consistent across the module and instead add error handling separately at a later stage if deemed necessary.